packages/cdktf-cli/lib/get/base.ts

```diff
@@ -101,5 +103,18 @@ export abstract class GetBase {
     }
   }
 
+  private static sanitizeSource(source: string, language: Language, isModule: boolean): string {
+    switch (language) {
+      case Language.JAVA:
+        return !isModule && source === "null" ? "nullprovider" : source.replace(/\//gi, '.').replace(/-/gi, '_');
+      case Language.CSHARP:
+        return !isModule && source === "null" ? "Providers.Null" : source.replace(/\//gi, '.').replace(/-/gi, '_');
```

Comment:
I think this would deserve a comment why the `null` provider is special cased here.